bundles/org.eclipse.equinox.p2.repository/src/org/eclipse/equinox/internal/p2/repository/messages.properties

```diff
@@ -80,4 +80,5 @@ connection_to_0_failed_on_1_retry_attempt_2=Connection to {0} failed on {1}. Ret
 UnableToRead_0_TooManyAttempts=Unable to read repository at: {0}. Too many failed login attempts.
 UnableToRead_0_UserCanceled=Unable to read repository at: {0}. Login canceled by user.
 RepositoryTransport_failedReadRepo=Error while reading from repository: {0}.
-unsafeHttp=Using unsafe http transport to retrieve {0}, see CVE-2021-41033. Consider using https instead.
\ No newline at end of file
+RepositoryTransport_unsafeHttp=Using unsafe http transport to retrieve {0}, see CVE-2021-41033. Consider using https instead.
+RepositoryTransport_unsafeHttpBlocked=Using unsafe http transport to retrieve {0} is blocked, see CVE-2021-41033. Only https is supported. Use the system property -Dp2.httpAction=redirect to automatically redirect to https or -Dp2.httpAction=allow to permit unsafe access.
```

Comment:
I would strip the "Only https is supported" as it is obviously supported but blocked as indicated by the first sentence.